django/test/client.py

```diff
@@ -651,12 +651,12 @@ def _parse_json(self, response, **extra):
             response._json = json.loads(response.content.decode(), **extra)
         return response._json
 
-    def _handle_redirects(self, response, **extra):
+    def _handle_redirects(self, response, data='', content_type='', **extra):
         """
         Follow any redirects by requesting responses from the server using GET.
         """
         response.redirect_chain = []
-        while response.status_code in (301, 302, 303, 307):
+        while response.status_code in (301, 302, 303, 307, 308):
```

Comment:
`http` contains these codes in constants. https://docs.python.org/3/library/http.html Would it be an idea to use them for readability?

Reply:
Yeah, good point, I'll change them